### PR TITLE
My first Microverse mobile project

### PR DESCRIPTION
**Pull request by bohaz**
Features Added
-Linters configuration
-Git ignore
-Html and CSS files
-Readme file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,180 @@
...
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+
+# 📖 [Hello Microverse project] <a name="about-project"></a>
+
```
Comment (tufoinnkuo10) [inline README.md:35]: - [optional] please always rename your project in the README to reflect what you are currently working on. you can rename this title like you did up.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@200&display=swap" rel="stylesheet">
+  <title>First mobile</title>
+</head>
+<body>
+  <header>
+    <div class="toolbar">
+      <a href="https://www.google.com">My Logo</a>
+      <button>
+        <img src="img/Union.png" alt="wite bars">
+      </button>
```
Comment (tufoinnkuo10) [inline index.html:22]: - [ ] please kindly ensure The hamburger button is on the right, and has a `nav` tag.

* The `nav` tag is for semantics reason, and the right position for convention and user experience for mobile devices.
* kindly remove the button tag from the hamburger icon
Screenshot (1021)_LI

File: index.html [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    </div>
+  </header>
+    <main>
+      <section class="head">
+        <div class="container">
+          <h1>Hey There.<br>I’m Ricardo.</h1>
+          <h2>I'm a Software Developer</h2>
+          <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+            don’t hesitate to contact me.</p>
+          <img src="img/Social Media.png" alt="social"/>
```
Comment (tufoinnkuo10) [inline index.html:32]: - [ ] Great job with the social icons. unfortunately its not an image but they a different social icons which are aimed at linking the viewer to your different social profiles. please the list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag).

* Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href.

File: style.css [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+  color: brown;
+}
+
+.toolbar {
+  display: flex;
+  flex-direction: row;
+  background-color: #3c3a39;
+  align-items: center;
+  align-content: space-between;
+  width: 422px;
```
Comment (tufoinnkuo10) [inline style.css:44]: - [ ] please set this width to 100% so as to make it responsive, also justify-content: space-between

File: style.css [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+  background-image: url(img/Group.png);
+  background-repeat: no-repeat;
+  background-position: 292px 60%;
+  background-color: #1c1a19;
+  display: flex;
+  flex-direction: column;
+  padding: 120px 24px;
+  gap: 12px;
+  position: absolute;
+  width: 375px;
```
Comment (tufoinnkuo10) [inline style.css:61]: - [ ] Remember you work should be responsive up to the desktop version (768px) kindly set the width here to 100% as well to make the section responsive, ensure the background image is well positioned after making the change
Screenshot (1022)_LI

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @bohaz 

There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally


Required Changes ♻️
Check the comments under the review

tufoin

Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by stanleySimeon — CHANGES_REQUESTED** [review]
# Hi @maths29 
 
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- Correct use of Gitflow ✔️
- Linter checks are passed ✔️
- Good PR description ✔️
- Meaningful commit messages ✔️

# Required Changes ♻️

* [ ] Kindly I suggest you follow the Figma design to make your project look more professional. Check this [link](templates in Figma) to follow the Figma template.

> ### The Figma design
>
>Screenshot 2023-04-30 at 00 02 39
>
> ### Your design
>
>Screenshot 2023-04-30 at 00 03 18


- [ ]  _Check the inline comments under the review for sections to improve._

### Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

Cheers, and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @stanleySimeon in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Ibizugbe — COMMENTED** [review]
Hi <STUDENT_MENTION>, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

I can see you are repeating the week and you are using the same PR from last week for same milestone, Kindly checkout to a new branch and ensure all the requirements for the milestone are met. open a PR fro the new branch and submit for review👍.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by bohaz** [comment]
Hi i have created a new pull request for this new week

**Pull request merged**